src/vertification.ts

```diff
@@ -0,0 +1,288 @@
+import fs from "node:fs";
+import dotenv from "dotenv";
+import { Client, EmbedBuilder, GuildMember, Message, TextChannel } from "discord.js";
+import { createTransport } from "nodemailer";
+import { createHash } from "crypto";
+import xlsx from "xlsx";
+const { readFile, writeFile, utils } = xlsx;
+import * as cron from "node-cron";
+
+dotenv.config(); // Load env parameters
+if (!process.env.DISCORD_GUILD_ID || !process.env.EMAIL_USERNAME || !process.env.EMAIL_PASSWORD || !process.env.EMAIL_HOST || !process.env.EMAIL_PORT || !process.env.VERIFICATION_CHANNEL) {
+    throw new Error("Environment tokens are not defined!");
+}
+
+const transporter = createTransport({
+    host: process.env.EMAIL_HOST,
+    port: +process.env.EMAIL_PORT,
+    auth: {
+        user: process.env.EMAIL_USERNAME,
+        pass: process.env.EMAIL_PASSWORD,
+    },
+});
+
+type Verification = {
+    name: string;
+    email: string;
+    discord_identifier: string;
+    payment_status: string;
+};
+type SpreadsheetRow = {
+    [key: string]: string | number;
+};
+
+export const members_to_monitor: Set<string> = new Set();
+const processing_members_code: Map<string, { email: string; id: string; time_stamp: number }> = new Map(); // Members and their codes
+const FILE_PATH = "~/Documents/onedrive-sync/Verification Team Roster.xlsx";
+const DEBUG_FILE_PATH = "./verification.xlsx"; // For debugging purposes only
```

Comment:
I told you that the file path should refer to inside the onedrive folder, as that is what will be mounted when the docker container is launched. Remember that the docker container is completely separate from the host unless you explicitly mount something onto the container. Because of this, it's not necessary to have a debug path, the debug and standard path can be the same, as when you mount to the docker container for production, you can specify which folder is getting mounted. See: https://github.com/GryphonRacingFSAE/Discord-Bot/blob/5ea070cc43feb320cb0d19850b76ddd15761799d/.github/workflows/CI.yml#L71C203-L71C203 and https://docs.docker.com/storage/volumes/

FILE_PATH should look like this: `./onedrive/Gryphon Racing Administrative/Team Roster/Verification Team Roster.xlsx` or whatever the current path is.
You also need to add the path to be mounted in the CI (the same line that I sent before).